pipe_segment/message_schema.py

```diff
@@ -144,3 +144,6 @@
         {"name": "frag_id", "type": "STRING", "mode": "NULLABLE"},
     ]
 )
+
+
+fieldnames = list(map(lambda x: x['name'], message_output_schema['fields']))
```

Comment:
For this immediately-realized collections, I prefer list comprehension syntax:

`fieldnames = [x['name'] for x in message_output_schema['fields']]`

I think it's clearer if you have to immediately realize the collection via `list(...)`, and also if the transformation is very simple like this one.

Up to you though.

Reply:
For me it's the same. But bear in mind this is not new code. It was just moved from somewhere else.